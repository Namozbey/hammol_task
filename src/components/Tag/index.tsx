import React, { ReactNode } from "react";
import styled from "styled-components";

interface Props {
  children: ReactNode;
}

const Wrapper = styled.div`
  border: 1px solid #979797;
  border-radius: 4px;
  padding: 4px 16px;
  font-family: Montserrat, sans-serif;
  font-size: 14px;
  font-style: normal;
  font-weight: 500;
  line-height: 17px;
  letter-spacing: 0px;
  text-align: center;
`;

export default function Tag(props: Props) {
  return <Wrapper>{props.children}</Wrapper>;
}
