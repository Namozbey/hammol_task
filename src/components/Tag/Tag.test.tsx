import React from "react";
import { render } from "@testing-library/react";
import Tag from ".";

describe("Testing Tag", () => {
  it("Snapshots", () => {
    const { asFragment } = render(<Tag>2016</Tag>);
    expect(asFragment()).toMatchSnapshot();
  });
});
