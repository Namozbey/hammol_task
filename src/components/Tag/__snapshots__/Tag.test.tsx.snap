// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Tag Snapshots 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ gJoQXV"
  >
    2016
  </div>
</DocumentFragment>
`;
